tests-examples/demo-todo-app.spec.ts

```diff
@@ -0,0 +1,437 @@
+import { test, expect, type Page } from '@playwright/test';
+
```

Comment:
Is this file needed? Seems like an example and not a true test

Reply:
Nope i will delete it